zk.js/src/test-utils/functionalCircuit.ts

```diff
@@ -70,15 +71,15 @@ export async function functionalCircuitTest(
   }
   await tx.compile();
 
-  await tx.getProof();
+  await tx.getProof(account);
   // unsuccessful proof generation
   let x = true;
 
   try {
     tx.proofInput.inIndices[0][1][1] = "1";
     // TODO: investigate why this does not kill the proof
     tx.proofInput.inIndices[0][1][0] = "1";
-    await tx.getProof();
+    await tx.getProof(account);
     x = false;
   } catch (error) {
     // assert.isTrue(error.toString().includes("CheckIndices_3 line:"));
```

Comment:
commented assert

Reply:
Good catch, the test would have failed if it didn't throw an error. The commented account was just the precise part of the check. 
I have added an equivalent check now.